frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -256,6 +267,58 @@ export const surveyLogic = kea<surveyLogicType>([
                 return { ...values.surveySingleChoiceResults, [questionIndex]: { labels, data, total } }
             },
         },
+        surveyMultipleChoiceResults: {
```

Comment:
Don't have to do this now but we could consider moving all this into its own results logic whenever we have a moment :) It'll make it easier to isolate and logic test too. The only mildly annoying part would be making sure to link that results logic to the correct survey, which is worthwhile to learn in kea if you haven't already yet ~